app/messaging/process-message.js

```diff
@@ -0,0 +1,41 @@
+const util = require('util')
+const { set } = require('../repositories/application-repository')
+const { applicationResponseMsgType, applicationResponseQueue } = require('../config')
+const sendMessage = require('../messaging/send-message')
+const createReference = require('../lib/create-reference')
+const { notify: { templateIdApplicationComplete } } = require('../config')
+const sendEmail = require('../lib/send-email')
+const boom = require('@hapi/boom')
+const processApplication = async (msg) => {
+  const reference = createReference()
+  await set({
+    reference,
+    type: 'VV001',
+    data: JSON.stringify(msg.body),
+    createdBy: 'admin',
+    updatedBy: 'admin',
```

Comment:
Should `updatedBy` be set when the application is first processed? I would have expected it to be left null.